hawkular-wildfly-monitor/src/main/resources/module/main/module.xml

```diff
@@ -23,6 +23,7 @@
     <resource-root path="hawkular-bus-rest-client.jar"/> <!-- DELETE THIS ONCE WE NO LONGER SEND TWO MESSAGES -->
     <resource-root path="hawkular-dmr-client.jar"/>
     <resource-root path="hawkular-feed-comm-api.jar"/>
+    <resource-root path="hawkular-inventory-json-helper.jar"/>
     <resource-root path="hawkular-monitor.jar"/>
     <resource-root path="inventory-api.jar"/>
```

Comment:
nope, `hawkular-inventory-json-helper` has the `inventory-api` as a transitive dependency, so there is no need to have both in `pom.xml`, but I guess, in `module.xml` all the jars need to be explicitly mentioned.